src/docs/reference/modules/Resources/README.md

```diff
@@ -365,6 +369,35 @@ The style block will only be injected once based on its name and can optionally
     }
 </style>
 ```
+#### Link tag
+
+A link tag is used to define the relationship between the current document and an external resource such as a a favicon or stylesheet. For a stylesheet, however, use the [style helper](#register-a-named-script). 
```

Comment:
a a favicon 

double a a

I can't edit this, or would have, but other than that looks good